engine/runtime/src/main/java/org/enso/interpreter/runtime/callable/argument/CallArgumentInfo.java

```diff
@@ -218,12 +218,13 @@ public FunctionSchema getPostApplicationSchema() {
           this.existingOversaturatedArgs.length,
           newOversaturatedArgInfo.length);
 
-      return new FunctionSchema(
-          originalSchema.getCallerFrameAccess(),
-          definitions,
-          argumentUsed,
-          oversaturatedArgInfo,
-          originalSchema.getAnnotations());
+      return FunctionSchema.newBuilder()
```

Comment:
Nicer. Have you refactored this manually or did an IDE help you?

Reply:
> did an IDE help you?

No. I didn't know that IntelliJ [supports this refactoring](https://www.jetbrains.com/help/idea/replace-constructor-with-builder.html), I will try to use it next time.